Src/Support/GoogleApis.Core/Apis/Util/RequestParameterAttribute.cs

```diff
@@ -64,13 +64,16 @@ public RequestParameterAttribute(string name, RequestParameterType type)
         }
     }
 
-    /// <summary>Describe the type of this parameter (Path or Query).</summary>
+    /// <summary>Describe the type of this parameter (Path, Query or UserDefinedQueries).</summary>
     public enum RequestParameterType
     {
         /// <summary>A path parameter which is inserted into the path portion of the request URI.</summary>
         Path,
 
         /// <summary>A query parameter which is inserted into the query portion of the request URI.</summary>
-        Query
+        Query,
+
+        /// <summary>A group of user-defined parameters that will be added in to the query portion of the request URI.</summary>

```

Comment:
[optional] I would also mention that in case this type is being used, the name of the RequestParameterAttirbute is meaningless.
What do you think?